test/support/test_config.h

```diff
@@ -106,7 +100,7 @@
 #define _PSTL_GLIBCXX_TEST_COMPLEX_PLUS_EQ_BROKEN _PSTL_GLIBCXX_TEST_COMPLEX_BROKEN
 #define _PSTL_GLIBCXX_TEST_COMPLEX_TIMES_EQ_BROKEN _PSTL_GLIBCXX_TEST_COMPLEX_BROKEN
 
-#define _PSTL_MSVC_LESS_THAN_CPP20_COMPLEX_CONSTEXPR_BROKEN (_MSC_VER && _ONEDPL_TEST___cplusplus <= 201703L)
+#define _PSTL_MSVC_LESS_THAN_CPP20_COMPLEX_CONSTEXPR_BROKEN (_MSC_VER && !(__cplusplus > 201703L || _MSVC_LANG > 201703L))

```

Comment:
It's better to make < 202002L (or whatever it is) check. Probably even better to use `TEST_VER` macro for that.

Reply:
Fixed.